<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-KOMPLEC" failures="1" tests="1" name="runner.FailedTestRunner" time="12.244" errors="0" timestamp="2024-09-08T11:39:07 IST" skipped="0">
  <testcase classname="runner.FailedTestRunner" name="runScenario" time="12.244">
    <failure message="expected [Warning: N1o match for E-Mail Address and/or Password.] but found [Warning: No match for E-Mail Address and/or Password.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Warning: N1o match for E-Mail Address and/or Password.] but found [Warning: No match for E-Mail Address and/or Password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at steps.LonginFunctionalitySteps.login_should_not_be_succeful_and_warning_messeage_displayed_as(LonginFunctionalitySteps.java:62)
at âœ½.login should not be succeful and warning messeage displayed as "Warning: N1o match for E-Mail Address and/or Password."(file:///D:/CucumberPractice/CucumberPracticeVersion/Features/LoginFunctioniality.feature:22)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- runner.FailedTestRunner -->
